### ISSUE TITLE
How to use a fixed-size crop-box?

### ISSUE THREAD

**Issue by derwaldgeist** [fengyuanchen/cropper#522]
I want to use this cropper for avatar creation. The size of the crop-box and the aspect ration should be fixed to 1:1 and 128x128 pixels, respectively. The image itself under the crop-box should be draggable (to place the desired cutout area below the crop-box), but not the crop-box itself.

Thus, I set-up the cropper with these options:
```js
    aspectRatio: 1 / 1,
    dragMode: 'move',
    cropBoxMovable: false,
    cropBoxResizable: false,
    minCropBoxWidth: 128,
    minCropBoxHeight: 128,
```
but the crop-box is larger than 128x128 pixels. I tried to set a maxCropBoxWidth and maxCropBoxHeight, but these option don't seem to be supported (and are not mentioned in the docs, anyway). Is there another way to fix the crop-box height to a certain size?

**Comment by fengyuanchen** [comment]
I suggest you just set the `aspectRatio` to `1`, don't mind the size the crop box as you can resize the cropped image to `128` on the server-side after crop it (or use the `getCroppedCanvas` method to crop and resize it).

**Comment by derwaldgeist** [comment]
Ah, I did not see that getCroppedCanvas also supports setting the width/height to new values. This is great, I used the loadImage() for this after-processing, so I can skip this step. And yes, after having implemented this, I agree that the user experience is even better now, since the crop box is larger which allows for a better positioning of the resulting window. BTW: Thanks for this great library, it has improved the usability of my app a lot!
